spec/requests/api/v1/digital_objects_spec.rb

```diff
@@ -1,9 +1,135 @@
 require 'rails_helper'
 
-RSpec.describe "Digital Objects API endpoint", type: :controller do
+RSpec.describe "Digital Objects API endpoint", type: :request do
+  let(:authorized_object) { FactoryBot.create(:digital_object_test_subclass, :with_sample_data, :with_lincoln_project) }
+  let(:authorized_project) { authorized_object.projects.to_a[0] }
```

Comment:
Can this be `authorized_object.projects.first`?

Reply:
Yes!